src/h3lib/lib/faceijk.c

```diff
@@ -747,8 +747,8 @@ void _faceIjkToCellBoundary(const FaceIJK *h, int res, int start, int length,
             adjacent hexagon edge will lie completely on a single icosahedron
             face, and no additional vertex is required.
             */
-            bool isIntersectionAtVertex =
-                _v2dEquals(&orig2d0, &inter) || _v2dEquals(&orig2d1, &inter);
+            bool isIntersectionAtVertex = _v2dAlmostEquals(&orig2d0, &inter) ||
```

Comment:
Seems like this remove the only two usage of `_v2dEquals`

So I guess you could either rename `_v2dAlmostEquals` or get rid of `_v2dEquals`.

Reply:
Good point, will drop the exact version